src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,177 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Avoid using wildcard imports. IntelliJ has a default number (>3 or >5) of classes before they convert your imports into wildcard imports, you can change this in your settings :)